utils/misc.go

```diff
@@ -76,3 +79,22 @@ func RandomID(randomIDLenth int) string {
 func UUID() string {
 	return uuid.New().String()
 }
+
+// GenRandNumber generates a random positive number between lower and upper.
+// The return value should be [lower, upper) and error is nil when success.
```

Comment:
```suggestion
// The return value should be between [lower, upper] and error is nil when success.
```

Reply:
I thought the random return would not include the upper number. e.g. rand.Int(io.Reader, 5) -> {0, 1, 2, 3, 4}.
So I use [lower, upper) for it. ref: https://pkg.go.dev/crypto/rand#Int